src/components/visit/visitList.tsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from 'react'
+import { getVisitData, Visit } from 'databaseIntigration'
+
+import VisitInstance from './visitInstance'
+
+interface VisitListProps {
+  searchQuery: string
+}
+
+export default function VisitList(props: VisitListProps) {
+  const [visitData, setVisitData] = useState<Visit[]>([])
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const response: Visit[] = await getVisitData()
+      setVisitData(response)
+    }
+    fetchData()
```

Comment:
Async await is generally a good idea, but sometimes using standard promises can be cleaner.

Consider:
```tsx
    const fetchData = () => getVisitData().then(setVisitData);
```

Reply:
Also note how I am passing setVisitData as a [first class function](https://developer.mozilla.org/en-US/docs/Glossary/First-class_Function) here